molecule/master_with_worker/converge.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+- name: Configure host
+  hosts: localhost
+  connection: local
+  become: yes
+  pre_tasks:
+    - name: Set proper nf_conntrack_max
+      ansible.posix.sysctl:
+        name: net.netfilter.nf_conntrack_max
+        value: '393216'
+        sysctl_set: true
+        state: present
+        reload: true
+
+- name: Configure instances
+  hosts: all
+  gather_facts: false
+  tasks:
+    - name: Install Python
+      ansible.builtin.raw: apt update && apt install -y python3-pip python3 python3-apt
+      changed_when: false
+
+- name: Converge
+  hosts: all
+  become: yes
+  become_method: su
+  gather_facts: false
+  pre_tasks:
+    - name: Install prerequisites
+      ansible.builtin.apt:
+        name:
+          - iproute2  # provides network facts
+          - kmod  # modprobe and lsmod
+          - net-tools
```

Comment:
Add curl installation into all test scenarios or at least check if curl is already installed